apps/web/src/design-system/table/Table.styles.ts

```diff
@@ -16,10 +16,12 @@ export default createStyles((theme: MantineTheme, _params, getRef) => {
       borderCollapse: 'collapse',
       borderSpacing: '0px 20px',
       'tr td:first-of-type': {
-        paddingLeft: 30,
+        paddingLeft: withSelection ? 10 : 30,
+        paddingRight: withSelection ? 10 : 30,
```

Comment:
the styles for the table when it has an option to select a row... more about it below